src/content/configuration/dev-server.md

```diff
@@ -604,6 +604,40 @@ webpack-dev-server --info=false
 ```
 
 
+## `devServer.injectClient`
+
+`boolean: false` `function (compilerConfig)`
+
+Tells `devServer` to inject a client. Setting `devServer.injectClient` to `true` will result in always injecting a client. It is possible to provide a function to inject conditionally:
+
+
+```javascript
+module.exports = {
+  //...
+  devServer: {
+    injectClient: (compilerConfig) => compilerConfig.name === 'only-include'
+  }
+};
+```
+
+
+## `devServer.injectHot`
+
+`boolean: false` `function (compilerConfig)`
+
+Tells `devServer` to inject a Hot Module Replacement. Setting `devServer.injectHot` to `true` will result in always injecting. It is possible to provide a function to inject conditionally:
+
+
+```javascript
+module.exports = {
+  //...
+  devServer: {
+    injectHot: (compilerConfig) => compilerConfig.name === 'only-include'
```

Comment:
Please add `hot` because `injectHot` is used when HMR is enabled.

Reply:
Great catch @hiroppy 
I've pushed an update